multi/stm32l4-multi/tty.c

```diff
@@ -42,6 +43,10 @@
 #define THREAD_STACKSZ 768
 #define THREAD_PRIO 1
 
+#if TTY_CNT != 0 && !BUILTIN_DUMMYFS
+#error "rootfs is needed when tty is enabled"
+#endif
```

Comment:
rootfs is not optional - being glued to the multidriver is. So if !BUILTIN_DUMMYFS we should rather wait for rootfs to register

Reply:
Ok, so I misunderstood the idea. If BUILTIN_DUMMYFS is not enabled, then we should assume that someone else (before multidrv can be operational) will create it?

Reply:
We need to wait for rootfs to register

Reply:
Done